sorted_words.rb

```diff
@@ -0,0 +1,18 @@
+class WordsList
+  def initialize
+    @words = []
+    while true
```

Comment:
This totally works, but a `while true` loop makes me kind of nervous because `true` will always be `true`. I know you are doing a `break` down below, but I think it might make sense to have a different check here for the loop to end.

You are tracking the words in an array (`@words`) so it feels a little safer to do something like

```rb
until @words.last == ''
  <do code>
end
```

Reply:
It's really helpful to know Ruby arrays have a .last method. I used `while true` only because I didn't know how to work around assigning '' to  create a `word` variable before running a loop which condition'd be `while word != ''`.